README.md

```diff
@@ -781,6 +781,9 @@ Flags:
   - :nerd_face: `--format=wide`: Wide table
   - :nerd_face: `--format=json`: Alias of `--format='{{json .}}'`
 - :whale: `--digests`: Show digests (compatible with Docker, unlike ID)
+- :whale: `-f, --filter`: Filter the images
+  - :whale: `--filter=before=<image:tag>`: Images created before given image (exclusive)
+  - :whale: `--filter=since=<image:tag>`: Images created after given image (exclusive)
```

Comment:
Could you document unimplemented filters such as “label”?

Reply:
Done